ci/test.sh

```diff
@@ -197,6 +195,22 @@ main() {
 
         rm -rf "${td}"
     fi
+
+    # tst cmake support
+    td="$(mkcargotemp -d)"
+
+    git clone --depth 1 https://github.com/cross-rs/rust-cmake-hello-world "${td}"
+
+    pushd "${td}"
+    retry cargo fetch
+    if (( ${STD:-0} )) && (( ${RUN:-0} )); then
```

Comment:
The `rust-cmake-hello-world` has both a `no_std` lib and a `std` binary, so we can check support for either/or.